numpy/random/src/distributions/distributions.c

```diff
@@ -957,6 +957,39 @@ RAND_INT_TYPE random_geometric(bitgen_t *bitgen_state, double p) {
   }
 }
 
+RAND_INT_TYPE random_two_sided_geometric(bitgen_t *bitgen_state, double alpha) {
```

Comment:
Should be int64_t.  No need for RAND_INT_TYPE since only available as part of Generator